biosynseq/generate_test.py

```diff
@@ -0,0 +1,11 @@
+"""Test module for generate.py."""
+
+from biosynseq.generate import fasta_to_embeddings, generate_fasta
+import numpy as np
+
+def test_generate_fasta() -> None:
+
+    return None
+
+def test_fasta_to_embeddings() -> np.ndarray:
+    assert type(fasta_to_embeddings())
```

Comment:
the type() function will return the type of the function return, what functionality are you attempting to test here?